### ISSUE TITLE
Broken links on the huggingface portal

### ISSUE THREAD

**Issue by kirianguiller** [Helsinki-NLP/OPUS-MT-train#51]
Hi everyone,

I just post a message to warn you that some models link are broken on the portal.

Here is a list of some of them : 
- https://huggingface.co/Helsinki-NLP/opus-mt-sla-en
- https://huggingface.co/Helsinki-NLP/opus-mt-en-es
- https://huggingface.co/Helsinki-NLP/opus-mt-es-ca

**Comment by jorgtied** [comment]
Ah - that's not good, of course. I'll have to find a way to avoid this. There are rather frequent updates for some models and I cannot keep all the old ones in place. How important are those links to the original MarianNMT model in the huggingface portal? My solution would be to simply take those links away or just link to the overall list of models for the same language pair. Would that be sufficient?

**Comment by kirianguiller** [comment]
Yes, I think your solution would be the best and the least troublesome. If you point to the repo that has all the last models, then people can choose by themselves.
Where can these last models be found ? The one referred in the Opus-MT-train are older than the ones referred on huggingface.
